spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -326,7 +326,8 @@
             create :efile_submission, tax_return: nil, data_source: create(
               :state_file_nj_intake,
               primary_first_name: "Grace",
-              primary_last_name: "Hopper"
+              primary_last_name: "Hopper",
+              primary_middle_initial: ""
```

Comment:
[DUST] Why do we need to add `primary_middle_initial` here if it's an optional field?

Reply:
To overwrite the base intake, which has a value for middle initial